components/dropdown/src/floatingUI.mjs

```diff
@@ -10,7 +10,35 @@ export default class AuroFloatingUI {
     this.keyDownHandler = null;
   }
 
+  mirrorSize(fullscreen) {
```

Comment:
fullscreen styling in javascript is necessary because there is a logic that manipulate bibContent's inline style on non-mobile screen, so style from css will get ignored.